WebApplication/Backend/Repositorys/FeedbackRepository.cs

```diff
@@ -1,24 +1,154 @@
+                    resultList.Add(entity);
+
+                }
+                connection.Close();
+                return resultList;
+
         }
 
         internal Feedback GetFeedbackById(string feedbackId)
         {
             throw new NotImplementedException();
         }
-        internal void SetFeedbackApprovedValue(Feedback feedback)
+
+        ////Vucetic Marija RA157/2017
+        /// <summary>
+        ///set value of atrribute Approved
+        ///</summary>
+        ///<returns>
+        ///true if sucessful,else false
+        ///</returns>
+        ///<exception>
+        ///if any exception occurs method will return null
+        ///</exception>
+        ///<param name="feedback"> Feedback type object
+        ///</param>
+        internal void SetFeedbackApprovedValue(FeedbackDTO feedback)
         {
-            throw new NotImplementedException();
+                string[] dateString = feedback.Date.ToString().Split(" ");
+                string[] partsOfDate = dateString[0].Split("/");
+                if (feedback.Approved)
+                {
+
+                    string sql1 = "REPLACE  into feedbacks(Text,Approved,Date,PatientId,SerialNumber)Values('" + feedback.Text + "','" + 0
+                        + "','" + partsOfDate[2] + "-" + partsOfDate[1] + "-" + partsOfDate[0] + " ','" + feedback.PatientId + " ','" + feedback.SerialNumber + "')";
+                    MySqlCommand cmd1 = new MySqlCommand(sql1, connection);
+                    cmd1.ExecuteNonQuery();
+             
+            }
+                else
+                {
+                    string sql1 = "REPLACE  into feedbacks(Text,Approved,Date,PatientId,SerialNumber)Values('" + feedback.Text + "','" + 1
+                        + "','" + partsOfDate[2] + "-" + partsOfDate[1] + "-" + partsOfDate[0] + " ','" + feedback.PatientId + " ','" + feedback.SerialNumber + "')";
+
+                    MySqlCommand cmd1 = new MySqlCommand(sql1, connection);
+                    cmd1.ExecuteNonQuery();
+              
+            }
+               connection.Close();
+        }
+
+
+        ////Repovic Aleksa RA52-2017
+        /// <summary>
+        ///Adds new row into feedbacks table
+        ///</summary>
+        ///<returns>
+        ///true if sucessful,else false
+        ///</returns>
+        ///<exception>
+        ///if any exception occurs method will return false
+        ///</exception>
+        ///<param name="feedback"> Feedback type object
+        ///</param>
+        internal bool AddNewFeedback(Feedback feedback)
+        {
+                string[] dateString = DateTime.Now.ToString().Split(" ");
+                string[] partsOfDate = dateString[0].Split("/");
+                string sqlQueryAdd = "INSERT INTO feedbacks (text,approved,date,patientid,serialnumber)  VALUES('" + feedback.Text + "','" + 0 + "','" + partsOfDate[2] + "-" + partsOfDate[1] + "-" + partsOfDate[0] + "T" + dateString[1]
```

Comment:
This name is a lot better, but this is actually not a query but dml 😄 Let's change that just in case